src/main/java/io/github/jhipster/online/service/GithubService.java

```diff
@@ -228,6 +228,7 @@ public void deleteAllOrganizationsUser(User user) {
      *  Sync the projects from the user's companies
      */
     private void syncUserProjectsFromCompanies(User user, GitHub gitHub, Set<GitCompany> currentGithubCompanies, Set<GitCompany> updatedGithubCompanies) throws IOException {
+        MDC.put("user.login", user.getLogin());
```

Comment:
Well, though I merged I would have rather used the interceptor approach as it would have worked globally because now you probably have a `User: null` in all logs when you are not going this path where you called `MDC.put`. However this would still be the same when application starts up for instance or for `@Scheduled` methods

Reply:
Ah, thanks. Let me try to do this in a separate PR. :smile:

Reply:
@gmarziou : Done at PR https://github.com/jhipster/jhipster-online/pull/190